ballerina/Module.md

```diff
@@ -74,23 +74,23 @@ This sample demonstrates a scenario of creating a secondary calendar and adding
 
 ### Step 1: Import the package
```

Comment:
In section,

Examples
The calendar connector provides practical [examples](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/tree/main/examples) illustrating usage in various scenarios. Explore these examples, covering use cases like creating calendar, scheduling meeting events, and adding reminders.


`calendar` should be `gcalendar`? if yes, that should be change in `Module.md`,`Package.md` as well.

Reply:
Fixed in [1b6905f](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/139/commits/1b6905f6c58710a688afb36dc7f054fe838f8843)